tests/devices/unit_tests/test_oav.py

```diff
@@ -68,7 +68,8 @@ def test_snapshot_trigger_saves_to_correct_file(
 ):
     image = PIL.Image.open("test")
     mock_save = MagicMock()
-    image.save = mock_save
+    # mypy doesn't like method assignments
+    image.save = mock_save  # type: ignore
```

Comment:
To prevent adding type: ignore, recommend using the context manager form of patching
https://github.com/python/mypy/issues/6713#issuecomment-620600319